### ISSUE TITLE
In powerbi, the custom visual from charticulator, the graph/chart does not change the size proportionally from all direction.

### ISSUE THREAD

**Issue by sujit1850** [microsoft/charticulator#317]
When imported, custom visual does not increase or decrease proportionally after one point of time. Rather than decreasing, the cropping effect takes place. One has to visit shapes section for changing the domain and range values. This is a manual setting. How about they behave just standard visuals whose size increase or decrease proportionally without being cropped.

Note: I have created the custom visual with one dataset and then using it for another dataset whose columns and values are entirely different.

**Comment by zBritva** [comment]
On export you have to enable auto domain and range for scales:

![image](https://user-images.githubusercontent.com/10897951/91549342-73937600-e92f-11ea-8eab-163be96b2e03.png)

**Comment by sujit1850** [comment]
I did this still the cropping effect is happening. 
[Chaticulator pages - proportion issue.docx](https://github.com/microsoft/charticulator/files/5188941/Chaticulator.pages.-.proportion.issue.docx)

I have attached word document enlisting all the steps with images. It has images of the issues. Let me know where I am going wrong. 
Also, I dont find much videos and tutorial on line for the key functionalities for powerbi.
